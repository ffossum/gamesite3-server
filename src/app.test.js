/* eslint-env jest */
const request = require('supertest');
const app = require('./app');

describe('GET / (unauthenticated)', () => {
  test('responds with html', async () => {
    await request(app.callback())
      .get('/')
      .expect(200)
      .expect('Content-Type', /html/)
      .expect(res => {
        expect(res.text).toMatchSnapshot();
      });
  });
});
