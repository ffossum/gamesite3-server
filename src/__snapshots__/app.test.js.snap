// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET / (unauthenticated) responds with html 1`] = `
"<!doctype html>

<html lang=\\"en\\">
<head>
  <meta charset=\\"utf-8\\">
  <title>Gamesite 3</title>

  
  <script src=\\"//localhost:8080/scripts/bundle.js\\" defer></script>
</head>

<body>
  <div id=\\"root\\"></div>
</body>
</html>
"
`;
